modules/administration-guide/partials/proc_disabling-readiness-init-containers-for-the-operator-installer.adoc

```diff
@@ -0,0 +1,25 @@
+[id="disabling-readiness-init-containers-for-the-operator-installer_{context}"]
+= Disabling the readiness init containers for the Operator installer
+
+To disable the previously enabled readiness init containers for {prod-short} installed by the Operator installer:
+
+.Prerequisites
+
+* {prod} installed by the Operator installer.
+
+.Procedure
+
+. Find {prod-short} Operator Deployment name.
```

Comment:
```suggestion
. Find the name of the {prod-short} Operator Deployment.
```

Reply:
@mmorhun, same comment as for `proc_disabling-`. Is it possible to be a bit more specific here re how to find it (maybe just expand the same sentence or add one more sentence)? Maybe a specific command can get the user this name?

Reply:
```suggestion
. Find the name of the {prod-short} Operator Deployment. Usually it is `{prod-id-short}-operator`:
```